core/src/main/java/io/kestra/core/repositories/ExecutionRepositoryInterface.java

```diff
@@ -142,6 +143,13 @@ Map<String, Map<String, List<DailyExecutionStatistics>>> dailyGroupByFlowStatist
         boolean groupByNamespaceOnly
     );
 
+    Map<String, ExecutionCountStatistics> executionCountsGroupByNamespace(
```

Comment:
```suggestion
    Map<String, ExecutionCountStatistics> executionCountsGroupedByNamespace(
```

Of maube just `executionCountsByNamespace`.